src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -0,0 +1,43 @@
+-- Enable RCSI
+ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON
+GO
+
+IF NOT EXISTS(SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'SchemaVersion' AND TABLE_SCHEMA = 'dbo')
+BEGIN
+    CREATE TABLE dbo.SchemaVersion (
+        [Version] int PRIMARY KEY, 
+        [Status] varchar(10) 
+    )
+END
+GO
+
+CREATE PROCEDURE dbo.SelectCurrentSchemaVersion
+AS BEGIN
+    SELECT MAX([Version])
+    FROM dbo.SchemaVersion 
+    WHERE [Status] = 'complete'
+END
+GO
+
+CREATE PROCEDURE dbo.UpsertSchemaVersion(
+        @version int,
+        @status varchar(10) 
+    )
+AS BEGIN
+    IF EXISTS(SELECT * FROM dbo.SchemaVersion WHERE [Version] = @version)
+    BEGIN
+        UPDATE dbo.SchemaVersion
+        SET [Status] = @status
+        WHERE [Version] = @version
+    END
+    ELSE
+    BEGIN
+        INSERT INTO dbo.SchemaVersion ([Version], [Status])
+        VALUES (@version, @status)
+    END
+END
+GO
+
+INSERT INTO dbo.SchemaVersion
```

Comment:
Looks like this is happening as well from the code. Seems we don't need this here.

Reply:
This is special casing for version 1.

Reply:
Sorry, I'm not following something. You call CompleteSchemaVersion() after ApplySchema() even for V1

Reply:
```
_schemaUpgradeRunner.ApplySchema(1);
```
vs
```
_schemaUpgradeRunner.InsertSchemaVersion(i);
_schemaUpgradeRunner.ApplySchema(i);
_schemaUpgradeRunner.CompleteSchemaVersion(i);
```